src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/WhoIsRegisteringType.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.overseasentitiesapi.model;
+
+public enum WhoIsRegisteringType {
+    AGENT("agent"),
+    SOMEONE_ELSE("someone_else");
+
+    private final String whoIsRegisteringType;
```

Comment:
This is one of the code smells. Could just remove 'Type' from the end of the name to make it consistent with other enums.